src/config.rs

```diff
@@ -518,6 +518,11 @@ pub struct Config {
     // TODO: DEPRECATED
     #[serde(default, deserialize_with = "failure_default")]
     unfocused_hollow_cursor: Option<bool>,
+
+    /// (Windows only) Enable experimental conpty backend
+    /// instead of using winpty. Only with Windows 10 Oct 2018 and later.
+    #[serde(default, deserialize_with="failure_default")]
+    enable_experimental_conpty_backend: bool
```

Comment:
Windows-only should be a `#[cfg(windows)]` annotation, not a comment.

Reply:
Thanks - I didn't know that `#[cfg(..)]` can be applied to individual struct fields!